### ISSUE TITLE
SQL Export of BIT fields is broken in 1.1

### ISSUE THREAD

**Issue by dmoagx** [sequelpro/sequelpro#2262]
(This is a different issue than #2254)

SQL-Export:
```sql
INSERT INTO `bits` (`id`, `bitf`)
VALUES
	(1,001),
	(2,000),
	(3,101),
	(4,010),
	(6,111),
	(8,011);
```

This format is syntactically wrong and will by MySQL interpreted as decimal numbers with leading zeros.
It should have been either `b'001'`,`b'101'`, and so on... or the real decimal values.

**Comment by hcouplet** [comment]
got the same issue with 1.1 where an export of a BIT(1) column was exported as 0000000.
Downgraded to 1.0.2 and everything was fine.

**Comment by robertoschwald** [comment]
Same issue here. But we can't use 1.0.2, as we are on El Capitan and ssh connections do not work with 1.0.2. Hope we get a new release asap.

**Comment by robertoschwald** [comment]
Works with nightly. Thanks for the fix.

**Comment by dmoagx** [comment]
Issue fixed in 1.1.1
See http://www.sequelpro.com/blog/2016.02/sequel-pro-1-1-1/
